### PR TITLE
Add newsletter July

### PR DESCRIPTION
**Pull request by duje**

### CODE REVIEW

**Comment by louwers** [comment]
You should really not add a 2MB image to the repo, git does not handle binary files well. There is also a limit of 2GB for repos which we will hit pretty quickly.

Also, I don't think our static site generator does any compression so it means everyone visiting the newsletter will have to load in this huge image as well. If you drag and drop the image to a GitHub editor it will get uploaded to GitHub's CDN and it will get compressed.

You can use this link but I would make a habit of it to do it with all images, especially large images.

https://github.com/maplibre/maplibre.github.io/assets/649392/a41976b9-a5e8-4995-a78c-2e20f832b3b0

Maybe you can move the Harfbuzz demo under MapLibre Native heading and mention it is a demo at this stage. Also you can mention Alan Wei Chen by name.

**Comment by louwers** [comment]
Not my topic, but I don't think we should ask people to follow an issue for news about the board elections. We can link to the issue that we are preparing it, but we should make a news post about it and update our socials when the details are known.

I think https://maplibre.org/maplibre-gl-js/docs/ are the new official MapLibre GL JS docs, it is not just a pre-release afaik.

File: content/news/2023-07-30-maplibre-newsletter-july-2023/index.md [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+
+Work on the Metal rendering backend has started, and the first important steps have been completed. The first one being: drawing nothing with Metal. And the second one being: drawing a triangle with Metal!
+
+<img src="https://user-images.githubusercontent.com/23352538/257043837-0dc17470-e1e7-49d5-8042-890482671023.png" style="max-width: 100%; height: auto;">
+
+(Screenshot from Tim Sylvester showing a triangle rendered with Metal.)
+
+## MapLibre GL JS
+
+A pre-release of the new documentation site can be found [here](https://maplibre.org/maplibre-gl-js/docs/).
```
Comment (HarelM) [inline content/news/2023-07-30-maplibre-newsletter-july-2023/index.md:54]: Not a pre-release, but rather a rewtire.

File: content/news/2023-07-30-maplibre-newsletter-july-2023/index.md [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+
+In addition, we released [v3.2.0](https://github.com/maplibre/maplibre-gl-js/releases/tag/v3.2.0) and [v3.2.1](https://github.com/maplibre/maplibre-gl-js/releases/tag/v3.2.1).
```
Comment (HarelM) [inline content/news/2023-07-30-maplibre-newsletter-july-2023/index.md:56]:
```suggestion
In addition, we released [v3.2.0](https://github.com/maplibre/maplibre-gl-js/releases/tag/v3.2.0) and [v3.2.1](https://github.com/maplibre/maplibre-gl-js/releases/tag/v3.2.1) with a lot of exciting new features and a lot of bug fixes. Click on the links above to see a detailed list of all the changes in each version.
```

**Review by HarelM — APPROVED** [review]

**Pull request merged**